app/src/main/java/org/mozilla/fenix/session/SessionNotificationService.kt

```diff
@@ -50,13 +50,23 @@ class SessionNotificationService : Service() {
                 metrics.track(Event.PrivateBrowsingNotificationTapped)
                 components.core.sessionManager.removeAndCloseAllPrivateSessions()
 
-                if (!VisibilityLifecycleCallback.finishAndRemoveTaskIfInBackground(this)) {
-                    startActivity(
-                        Intent(this, HomeActivity::class.java).apply {
-                            this.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
-                        }
-                    )
+                val homeScreenIntent = Intent(this, HomeActivity::class.java);
```

Comment:
Please remove the `;` at the end, this is another example where you could benefit from using the ktlint detection tool